platform/ios/CHANGELOG.md

```diff
@@ -4,14 +4,22 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 ## master
 
-### Annotations and user interaction
+### Styles and rendering
 
-* Increased the default maximum zoom level from 20 to 22. ([#9835](https://github.com/mapbox/mapbox-gl-native/pull/9835))
+* Added `MGLComputedShapeSource` source class that allows applications to supply vector data on a per-tile basis.
+* Properties such as `MGLSymbolStyleLayer.iconAllowsOverlap` and `MGLSymbolStyleLayer.iconIgnoresPlacement` now account for symbols in other sources. ([#10436](https://github.com/mapbox/mapbox-gl-native/pull/10436))
+* Improved the reliability of collision detection between symbols near the edges of tiles, as well as between symbols when the map is tilted. It is no longer necessary to enable `MGLSymbolStyleLayer.symbolAvoidsEdges` to prevent symbols in adjacent tiles from overlapping with each other. ([#10436](https://github.com/mapbox/mapbox-gl-native/pull/10436))
+* Symbols can fade in and out as the map pans, rotates, or tilts. ([#10436](https://github.com/mapbox/mapbox-gl-native/pull/10436))
```

Comment:
@ChrisLoer, did I do justice to #10436? Feel free to amend these blurbs as needed.

Reply:
@1ec5 That seems pretty reasonable. Looks like it hits the main points from the GL JS changelog (but translated into iOS-ese):

*Improve label collision detection
**Labels from different sources will now collide with each other
**Collisions caused by rotation and pitch are now smoothly transitioned with a fade
**Improved algorithm for fewer erroneous collisions, denser label placement, and greater label stability during rotation